src/app/component/ProductGallery/ProductGallery.component.js

```diff
@@ -9,16 +9,19 @@
  * @link https://github.com/scandipwa/base-theme
  */
 
-import { PureComponent, Fragment } from 'react';
+import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import Slider from 'Component/Slider';
-import Image from 'Component/Image';
-import './ProductGallery.style';
+import { TransformWrapper } from 'react-zoom-pan-pinch';
 import ProductGalleryAdditionalMedia from 'Component/ProductGalleryAdditionalMedia';
```

Comment:
Could you please rename this into `ProductGalleryThumbnailImage` and the `ProductGalleryImage` to `ProductGalleryBaseImage` this will then fit image names properly, might be less confusing.

When doing this, remember, that name of the file is it's component block prop value => name of the base class must also change.